src/inverseRobot.js

```diff
@@ -8,6 +8,29 @@
 
 function inverseRobot(robot) {
   // write code here
+  let count = 0;
+  const k = [];
+
+  for (const key in robot) {
+    count++;
+    k.push(key);
+  }
+
+  for (let i = 0; i < count; i++) {
```

Comment:
```suggestion

```
You dont need two loops here, you can do this if you cheking some indexes :)

Reply:
but without two loops how can I compare each pair in the object with each and every other pair?